drafts/use-cases/0.intro.md

```diff
@@ -48,31 +48,31 @@ These would be:
         },
         "operations": {
             "@id": "hydra:operation",
-            "@type": "hydra:Operation"
+            "@type": "@id"
         },
         "method": {
             "@id": "hydra:method",
             "@type": "xsd:string"
         },
         "returns": {
             "@id": "hydra:returns",
-            "@type": "hydra:Class"
+            "@type": "@id"
         },
         "expects": {
             "@id": "hydra:expects",
-            "@type": "hydra:Class"
+            "@type": "@id"
         },
         "supportedClasses": {
```

Comment:
Please change this to `supportedClass` (singular) to just be a shortcut and not an alias